test/testcanvas.go

```diff
@@ -183,6 +183,7 @@ func layoutAndCollect(objects []fyne.CanvasObject, o fyne.CanvasObject, size fyn
 	objects = append(objects, o)
 	if w, ok := o.(fyne.Widget); ok {
 		r := w.CreateRenderer()
+		r.Refresh()
```

Comment:
This line seems curious. Why was a renderer needing to be refreshed immediately after being created?

Reply:
Because the toolbar renderer was strange. The objs weren't set before a refresh.
I changed this with the following commit but left the refresh here. It can be removed now, I think.